root/static/scripts/relationship-editor/common/multiselect.js

```diff
@@ -95,7 +95,9 @@ import deferFocus from '../../edit/utility/deferFocus';
 
                 var visible = matchIndex(option, term) >= 0 && (
                     linkedEntities.link_attribute_type[typeGID].creditable ||
-                    _.findIndex(selected, function (a) { return a.type.gid === typeGID }) < 0
+                    _.findIndex(selected, function (a) {
+                        return a.type.gid === typeGID;
+                    }) < 0
```

Comment:
I'd personally prefer to move both selected and function into their own lines here, if you're ok with it? 

```
                    _.findIndex(
                        selected,
                        function (a) {
                            return a.type.gid === typeGID;
                        },
                    ) < 0
```

I feel it makes the comparison and whatnot clearer.

I might also move the closing ); to be just after the `< 0`, if eslint is happy with that.